### ISSUE TITLE
Warning Tracking

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/VideoDeduplication#14]
This issue will track common warnings that pop up. We can break critical ones out to their own task

**Comment by johnhbenetech** [comment]
At the end of extract_features.py:

For each video analyzed:
```
Cant load feature file /datadrive/representations/frame_level/mK1Nu8A1jEo.mp4_vgg_frames.npy
Cannot cast ufunc 'subtract' output from dtype('float64') to dtype('uint8') with casting rule 'same_kind'
```

Last output:
`WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/siamese_net.py:91: The name tf.summary.histogram is deprecated. Please use tf.compat.v1.summary.histogram instead.`

`WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/siamese_net.py:43: The name tf.train.Saver is deprecated. Please use tf.compat.v1.train.Saver instead.`

**Comment by johnhbenetech** [comment]
`WARNING:tensorflow: The TensorFlow contrib module will not be included in TensorFlow 2.0.`


`WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/model_tf.py:78: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.`


```
WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/model_tf.py:132: to_float (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.cast` instead.

```

`WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/nets/vgg.py:145: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.`

```
WARNING:tensorflow:From /anaconda3/envs/winnow-gpu/lib/python3.6/site-packages/tensorflow_core/contrib/layers/python/layers/layers.py:1057: Layer.apply (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.
Instructions for updating:
Please use `layer.__call__` method instead.
```
`WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/model_tf.py:114: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.`

`WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/model_tf.py:117: The name tf.global_variables_initializer is deprecated. Please use tf.compat.v1.global_variables_initializer instead.`

`WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/model_tf.py:96: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.`

`WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/model_tf.py:99: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.`

`WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/siamese_net.py:7: The name tf.logging.set_verbosity is deprecated. Please use tf.compat.v1.logging.set_verbosity instead.`

`WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/siamese_net.py:7: The name tf.logging.WARN is deprecated. Please use tf.compat.v1.logging.WARN instead.`

`
WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/siamese_net.py:36: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.`

`WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/siamese_net.py:91: The name tf.summary.histogram is deprecated. Please use tf.compat.v1.summary.histogram instead.
`
`WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/siamese_net.py:105: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.`

`WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/siamese_net.py:43: The name tf.train.Saver is deprecated. Please use tf.compat.v1.train.Saver instead.`

`WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/siamese_net.py:63: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.`

`WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/siamese_net.py:65: The name tf.global_variables_initializer is deprecated. Please use tf.compat.v1.global_variables_initializer instead.`

`WARNING:tensorflow:From /VideoDeduplication/winnow/feature_extraction/siamese_net.py:66: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.`

**Comment by johnhbenetech** [comment]
During extract features:
Trun track id unknown, no tfhd was found. [mov,mp4,m4a,3gp,3g2,mj2 @ 0x563c1a2cf640] error reading header
